app/javascript/packs/searchParams.js

```diff
@@ -176,6 +180,12 @@ function search(query, filters, sortBy, sortDirection) {
     }
   });
 
+  // Run Algolia code only if the ID is live.
+  if (document.body.dataset.algoliaId.length > 0) {
```

Comment:
```suggestion
  if (document.body.dataset.algoliaId?.length > 0) {
```
additional safety net if `algoliaId` is undefined for some reason.